Objects/methodobject.c

```diff
@@ -284,11 +295,11 @@ PyTypeObject PyCFunction_Type = {
     0,                                          /* tp_as_mapping */
     (hashfunc)meth_hash,                        /* tp_hash */
     PyCFunction_Call,                           /* tp_call */
-    0,                                          /* tp_str */
+    (reprfunc)meth_str,                         /* tp_str */
```

Comment:
Just use `meth_repr`.

Reply:
The point of the change is to have a `__str__` that’s different than `__repr__` :)  But see the ticket, I think I will scale back this PR to classes only.

Reply:
Oh well, it is Guido who wanted a unified behaviour for modules, classes and functions (since they are things created by syntax (import*, class and def) with a `__name__` attribute).

* I think a module’s `__name__` depends on its loader, not on the name used in the import statement (import as doesn’t change `__name__`)